server/express.js

```diff
@@ -1,16 +1,21 @@
 const express = require('express');
 const graphqlHTTP = require('express-graphql');
 const { buildSchema } = require('graphql');
+require('.envDOT')
```

Comment:
```suggestion
require('dotenv').config()
```

Reply:
Oh yeah. I just threw that seed together. I guess we gotta get that seeded with kinex tomorrow